resolvers/auth.py

```diff
@@ -110,6 +110,7 @@ def generate_unique_slug(src):
 
 @mutation.field("registerUser")
 async def register_by_email(_, _info, email: str, password: str = "", name: str = ""):
+    email = email.lower()
```

Comment:
do you want to include these changes here?

Reply:
yes, i'm a lazy ass :)